services/base-json.js

```diff
@@ -7,22 +7,13 @@ const trace = require('./trace')
 const { InvalidResponse } = require('./errors')
 
 class BaseJsonService extends BaseService {
-  async _requestJson({ schema, url, options = {}, errorMessages = {} }) {
+  _parseJson(buffer) {
```

Comment:
Extract `_parseJson()` helper function so it can be used in `fetchJsonFromRepo()`.

Reply:
Makes sense. Does this warrant an aside/footnote on the Badge Tutorial doc? The vast majority of services won't need to use the newly extracted `_parseJson` function  so I'm wondering what the potential is (if any) for a user working on a service to be confused and/or attempt to use `_parseJson` directly (when they really just need to use `_requestJson`)

Reply:
I imagine a developer reading this file probably would notice the `_requestJson` method too.